src/Features/VisualBasic/Portable/ExtractMethod/VisualBasicSelectionValidator.vb

```diff
@@ -494,6 +494,18 @@ result.ReadOutside().Any(Function(s) Equals(s, local)) Then
                        }
             End If
 
+            Dim commonRootContainer = commonRoot.FirstAncestorOrSelf(Of SyntaxNode)(Function(node) node.Kind = SyntaxKind.ClassBlock Or node.Kind = SyntaxKind.StructureBlock Or node.Kind = SyntaxKind.ModuleBlock)
+
+            If (commonRootContainer Is Nothing) Then
```

Comment:
```suggestion
            If commonRootContainer Is Nothing Then
```

Reply:
oops

Reply:
Show me a C# developer, and I'll show you a developer who puts parentheses around their if statements in VB.